src/directives/accessibility/splitter/splitter-accessibility.directive.ts

```diff
@@ -217,7 +217,7 @@ export class SplitterAccessibilityDirective implements OnInit, AfterViewInit, On
             const areas = this.getAreasFromGutter(event.target as HTMLElement);
 
             // set the previous area to it's minimum size
-            const delta = areas.previous.size - areas.previous.minSize;
+            const delta = areas.previous.size as number - areas.previous.minSize;
```

Comment:
Casting this as a number may not be the best approach here. The reason you had to add that is because the size may have a value of `*` or be a number. We should probably check if the value is `*` and handle that - perhaps we can't do anything in that case, but trying to do `'*' - areas.previous.minSize` will probably cause some issues if it were to occur.